xmipp

```diff
@@ -134,7 +135,7 @@ int main(int argc, char** argv){
 
     if (shrt==1)
     {
-        std::cout << "%s: %s" << std::endl;
+        std::cout << "%(VERSION_TAG)s: %(XMIPP_VERSION)s" << std::endl;
```

Comment:
Don't you miss some reference to `compilingInfo `?
This would be good candidate for str.format()

Reply:
I think not. I've double checked and pyCharm is saying that looks fine. Why you think so?

I did this before you seminar today. I even prefer the
```
>>> myVar1 = "some variables"
>>> myVar2 = "scope"
>>> print(f"Some string made by {myVar1} in the {myVar2}")
Some string made by some variables in the scope
```
but it was not totally clear to me if it's a good way or not..

Reply:
format is not a good choice here since this string contains several '{' and '}', then it should be replaced by '\{' and '\}', which is not desirable...